### PR TITLE
Updating the Superset icon to an OL branded logo

### PR DESCRIPTION
**Pull request by quazi-h**
Adding a local static image with the OL brand icon.
Updated superset_config to point to new app logo.

### What are the relevant tickets?
https://github.com/mitodl/ol-infrastructure/issues/1968

### Description (What does it do?)
Updates the app icon in the Superset UI.

### CODE REVIEW

File: src/ol_superset/pythonpath/superset_config.py [hunk 1]
```diff
@@ -35,7 +35,7 @@
 # --------------------------------
 APP_NAME = "MIT OL Business Intelligence"
 # Specify the App icon. Useful for white-labeling
-APP_ICON = "/static/assets/images/superset-logo-horiz.png"
+APP_ICON = "../static/assets/images/OL_sub-brand_lockup_one-line_rgb_black.png"
```
Comment (pdpinch) [inline src/ol_superset/pythonpath/superset_config.py:38]: We'll get a version of this that says "Data Platform" above "Open Learning" 

@quazi-h does it have to be a .png, or would a .svg work?

Comment (quazi-h) [reply src/ol_superset/pythonpath/superset_config.py:38]: Yes, I believe an .svg is an accepted icon format here

File: src/ol_superset/static/assets/images/OL_sub-brand_lockup_one-line_rgb_black.png
Comment (blarghmatey) [inline src/ol_superset/static/assets/images/OL_sub-brand_lockup_one-line_rgb_black.png:1]: Can you rename this to something a bit less verbose? Maybe just `ol-data-platform-logo.png`

File: src/ol_superset/pythonpath/superset_config.py (hunk 1 above)
Comment (blarghmatey) [inline src/ol_superset/pythonpath/superset_config.py:38]: I think this gets rendered as the path in the web UI for loading the image, so it shouldn't have the leading `../`. You'll also need to add a `COPY` command in the Dockerfile to place the logo into the final image at the proper location.

**Review by blarghmatey — CHANGES_REQUESTED** [review]

**Comment by pdpinch** [comment]
@quazi-h please use the attached svg logo:

DataPlatform-OL_sub-brand

File: src/ol_superset/Dockerfile [hunk 2]
```diff
@@ -1,6 +1,7 @@
 FROM apachesuperset.docker.scarf.sh/apache/superset:${SUPERSET_TAG:-latest}
 COPY --link requirements.in requirements.txt
 COPY --link pythonpath /app/pythonpath
+COPY --link static /app/static
```
Comment (blarghmatey) [inline src/ol_superset/Dockerfile:4]: This completely overrides the static directory in the docker image, which isn't what we want. Instead we just want to copy the file into the target directory.
```suggestion
COPY static/assets/images/ol-data-platform-logo.svg /app/static/assets/images/ol-data-platform-logo.svg
```

**Review by blarghmatey — APPROVED** [review]

**Pull request merged**